### PR TITLE
Replace console.logs with logging logs

### PR DESCRIPTION
**Pull request by naclcaleb**
Not sure if I put these in all the write places. 

I didn't know what to do with logs with a second parameter (i.e., `console.log("Something went wrong", err)`), so I just left them.

### CODE REVIEW

**Comment by naclcaleb** [comment]
Addresses #367

File: music/src/core/recorder.ts [hunk 1]
```diff
@@ -135,7 +137,7 @@ export class Recorder {
   }
 
   private midiReady(midi: WebMidi.MIDIAccess) {
-    console.log('Initialized Recorder');
+    logging.log('Initialized Recorder');
```
Comment (notwaldorf) [inline music/src/core/recorder.ts]: I would use "Recorder" as the prefix here (and everywhere where the prefix could be helpful)

File: music/src/core/soundfont.ts [hunk 2]
```diff
@@ -194,7 +195,7 @@ export class Instrument {
       sampleNamesAndURLs.forEach(
           (nameAndURL) => this.buffers.add(nameAndURL.name, nameAndURL.url));
       await new Promise(resolve => Tone.Buffer.on('load', resolve));
-      console.log(`Loaded samples for ${this.name}.`);
+      logging.log(`Loaded samples for ${this.name}.`);
```
Comment (notwaldorf) [inline music/src/core/soundfont.ts]: I'd probably use SoundFont here as the prefix

Comment (naclcaleb) [reply music/src/core/soundfont.ts]: Should I use that prefix for all the logs in the file?

Comment (notwaldorf) [reply music/src/core/soundfont.ts]: yeah, i think that would be neat (it helps see where the logs are coming from)

Comment (naclcaleb) [reply music/src/core/soundfont.ts]: Ok, I added a new commit.

Should I add this to the other files as well?

Comment (notwaldorf) [reply music/src/core/soundfont.ts]: yes please!

Comment (naclcaleb) [reply music/src/core/soundfont.ts]: Ok, does my latest commit look ok?

**Comment by naclcaleb** [comment]
Ok, I finally got it to pass the checks. Is this satisfactory?

File: music/src/core/soundfont.ts [hunk 3]
```diff
@@ -169,7 +170,7 @@ export class Instrument {
         samples
             .filter((info) => {
               if (info.pitch < this.minPitch || info.pitch > this.maxPitch) {
-                console.log(
+                logging.log(
                     `Pitch ${info.pitch} is outside the valid range for ${
                         this.name}, ignoring.`);
```
Comment (notwaldorf) [inline music/src/core/soundfont.ts]:
```suggestion
                        this.name}, ignoring.`, 'SoundFont');
```

**Comment by naclcaleb** [comment]
Ah, whoops! Thanks for the suggestion; I committed it. But now there's an error about Google CLA...

**Comment by notwaldorf** [comment]
No worries, it's just mad at me not you :)

**Comment by notwaldorf** [comment]
@googlebot I fixed it.

**Pull request merged**